### ISSUE TITLE
aria-labels in footer are too verbose

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/orangelight#3137]
**Expected behavior**
When the twitter, facebook, govdocs, and friends logos are given focus, a screen reader will read a concise description of the link, e.g. "PUL Twitter", "PUL Facebook", "Government Documents", "Friends of PUL".

These aria-labels are essentially acting as alt text, which [should be succinct](https://webaim.org/techniques/alttext/).

**Actual behavior**
The descriptions are not concise: "Opens Princeton University Library Facebook page", "Opens Princeton University Library Twitter page", "Opens Princeton University Library Government Documents Depository page", and "Opens Princeton University Library  Friends of the Library page".  Mentioning that it opens the particular page is redundant, since the screen reader will already tell users that it is a link.

**Steps to replicate**

1. Turn on voiceover
2. Press Ctl + Option + U to open the rotor
3. Use the Right/Left arrows to get the list of links
4. Use the Up/Down arrows to locate these links.  Note that the link text is quite verbose.
5. Press enter when one of the links is selected to set focus to it.  Note that it reads all the link text, plus mentioning that it is a link.

**Impact of this bug**
Screenreader experience is verbose and redundant.
